fractions/keycloak-microprofile-jwt/src/main/resources/modules/keycloak/io/smallrye/jwt/main/module.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.3" name="keycloak.io.smallrye.jwt">
+    <resources>
+      <artifact name="org.keycloak:keycloak-core:${version.keycloak}" />
+      <artifact name="org.keycloak:keycloak-common:${version.keycloak}" />
+      <artifact name="org.keycloak:keycloak-adapter-core:${version.keycloak}" />
+      <artifact name="org.keycloak:keycloak-admin-cli:${version.keycloak}" />
```

Comment:
This looks wrong. There should never be more than 1 module that uses a JAR as its resource root IMHO (otherwise you'll get those weird classloading errors). There are already `org.keycloak` modules that include these artifacts, so this one (which BTW has a very strange name) should have a dependency on them.

Reply:
sure let me check...

Reply:
it worked for the jackson artifacts but not for the keycloak ones, I've tried to use the modules as dependencies instead by bringing the [keycloak adapter feature pack](https://github.com/keycloak/keycloak/tree/master/distribution/feature-packs/adapter-feature-pack) in the pom and using the relevant modules.
My only explanation is that it works for Jackson because it is used indirectly by the KC factory while the classes referenced from the KC factory require an artifact-level support, possibly influenced by the fact ServiceLoader is involved... I'm not too concerned about using the artifacts instead of the modules at the moment.